### ISSUE TITLE
Bug: Collapsible Block needs full focus outline

### ISSUE THREAD

**Issue by annelfitz** [Esri/calcite-app-components#308]
### Summary
Currently the collapsible block focus outline is not showing up when it is collapsed. You can still use the "return" button to expand the content, but it is not showing a focus outline. After expanding it, there is just one line of the focus outline that is seen - shown in the following image: 
<img width="328" alt="Screen Shot 2019-09-27 at 10 59 18 AM 1" src="https://user-images.githubusercontent.com/51344676/65791203-1b983200-e116-11e9-9864-35103303fde3.png">

Tao recommends setting the `outline-offset` on the button - because this is not currently set.

### Actual Behavior
No focus outline on the collapsed block

### Expected Behavior
A focus outline on the collapsed block

### Reproduction Steps

1. https://esri.github.io/calcite-app-components/demos/calcite-block.html

**Comment by jimmieego** [comment]
Related: https://github.com/Esri/calcite-app-components/issues/253

**Comment by driskull** [comment]
This is because the `calcite-block` has `overflow:hidden`. Removing that fixes it and doesn't seem to cause any issues. @jcfranco can you confirm if that would be ok to do?

**Comment by jcfranco** [comment]
Installed.

**Comment by alocke** [comment]
@jcfranco I don't think you meant to assign this to me.

**Comment by jcfranco** [comment]
@alocke My bad! 😅

**Comment by annelfitz** [comment]
Looks good - but I am noticing this one small thing: there is a bit of a flicker of the outline when you open the block, not sure if this is expected or not
![calcite_block_flicker](https://user-images.githubusercontent.com/51344676/66139646-a2855880-e5b5-11e9-9584-23c66b93a3ec.gif)

**Comment by jcfranco** [comment]
@asangma Can you take a look at this? ☝️👀

**Comment by asangma** [comment]
The flicker is due to the animation on `.content`.
Fix installed. @annelfitz :)

**Comment by annelfitz** [comment]
Looks good now!

**Comment by jcfranco** [comment]
Closing as verified based on ☝️. Thanks @annelfitz!
